apps/payments/next/app/[locale]/[offeringId]/checkout/[interval]/[cartId]/start/page.tsx

```diff
@@ -19,12 +24,28 @@ export default async function Checkout() {
   const locale = headers().get('accept-language') || DEFAULT_LOCALE;
   const sessionPromise = auth();
   const l10nPromise = app.getL10n(locale);
-  const [session, l10n] = await Promise.all([sessionPromise, l10nPromise]);
+  const cartPromise = getCartAction(params.cartId);
+  const [session, l10n, cart] = await Promise.all([
+    sessionPromise,
+    l10nPromise,
+    cartPromise,
+  ]);
+
+  // Temporary function used to test handleStripeErrorAction
+  // This is to be deleted as part of FXA-8850
+  async function onSubmit() {
+    'use server';
+    const error: Stripe.StripeRawError = new Error(
+      'Whoops'
+    ) as unknown as Stripe.StripeRawError;
+    error.type = 'card_error';
+    await handleStripeErrorAction(cart.id, cart.version, error);
+  }
 
   return (
     <>
       <section
-        className="h-[640px] flex flex-col items-center justify-center"
+        className="h-min-[640px] flex flex-col items-center justify-center"
```

Comment:
Q: Why was this changed?

Reply:
The content of this section had gotten bigger than 640px and was overflowing.

We could actually remove this completely. It was initially only added to give the empty page some structure.